scripts/update_imagery.js

```diff
@@ -1,14 +1,31 @@
 /* eslint-disable no-console */
 const fs = require('fs');
-let sources = require('editor-layer-index/imagery.json');
 const prettyStringify = require('json-stringify-pretty-compact');
 
+/** @type {import("geojson").FeatureCollection} */
+const sources = JSON.parse(
+  fs.readFileSync(require.resolve('editor-layer-index/imagery.geojson'), 'utf8')
```

Comment:
I had to change `require` to `JSON.parse(fs.readFile(...))` because ELI uses the file extension `.geojson` instead of `.geo.json`, so it can't be imported